Specs/Number/Japanese/NumberModel.json

```diff
@@ -1,12 +1,12 @@
 [
   {
     "Input": "西九条",
-    "Comment": "PendingValidation",
+    "NotSupported": "dotnet, javascript, python, java",
```

Comment:
Please do not remove spec comments unless they no longer apply. Do all these mean the input sentences and any results were reviewed and are correct?